docs/product-installation-guide/src/main/asciidoc/dev-studio-con.adoc

```diff
@@ -1,6 +1,6 @@
 [id='dev-studio-con']
 = Red Hat JBoss Developer Studio 
 
-Red Hat JBoss Developer Studio is an integrated development environment (IDE) based on Eclipse. It integrates tooling and runtime components by combining Eclipse, Eclipse Tooling, and {EAP}. Red Hat JBoss Developer Studio provides plug-ins with tools and interfaces for {PRODUCT}. These plugins are based on the community version of these products. For this reason, the {Product} plug-in is called the Drools plug-in.
+Red Hat JBoss Developer Studio is an integrated development environment (IDE) based on Eclipse. It integrates tooling and runtime components by combining Eclipse, Eclipse Tooling, and {EAP}. Red Hat JBoss Developer Studio provides plug-ins with tools and interfaces for {PRODUCT}. These plug-ins are based on the community version of these products. For this reason, the {Product} plug-in is called the Drools plug-in.
 
 For more information about Red Hat JBoss Developer Studio, see <<dev-studio-install-proc>>.
```

Comment:
Not in this PR, but in 4.2. ACTIVATING ACTIVEMQ JMS BROKER, adjust prerequisites. Needs to be plural, bulleted, and each grammatically consistent (both should be passive). Also food for thought, the prereqs all around seem to be a tad inconsistent. Some say "X installed on ..." when should be "X is installed on ...", others are active while others passive, and some are just noun phrases instead of passive actions ("An NFS cluster" instead of "An NFS cluster is configured" or something). You probably remember our convo about prereqs style. Just pointing it out here.